### PR TITLE
Use new docs URL

### PR DESCRIPTION
**Pull request by vdiez**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,4 +45,5 @@ Desktop.ini
 .scannerwork
 
 # eslint-plugin-sonarjs
-eslint-plugin-sonarjs-*.tgz
\ No newline at end of file
+eslint-plugin-sonarjs-*.tgz
+its/eslint-plugin-sonarjs/package-lock.json
```
Comment (yassin-kammoun-sonarsource) [inline .gitignore:49]: Any reason why you want to gitignore this lock file?

File: packages/jsts/tests/rules/index.test.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ describe('index', () => {
       }
       expect(configs.recommended.plugins!['sonarjs'].rules).toHaveProperty(eslintId);
       expect(rule.meta.docs!.url).toBe(
-        `https://github.com/SonarSource/rspec/blob/master/rules/${sonarKey}/javascript/rule.adoc`,
+        `https://sonarsource.github.io/rspec/#/rspec/${sonarKey}/javascript`,
```
Comment (yassin-kammoun-sonarsource) [inline packages/jsts/tests/rules/index.test.ts:47]: Should we also update the `newRule` script to automatically include the URL to the rule rspec?

**Review by yassin-kammoun-sonarsource — APPROVED** [review]
Looks good to me, but I left two questions for you before merging :)

File: .gitignore (hunk 1 above)
Comment (vdiez) [reply .gitignore:49]: it was failing to install the tarball of the integration test during ci with the lock file. i don't know the reason, but without the lock file it works

File: packages/jsts/tests/rules/index.test.ts (hunk 2 above)
Comment (vdiez) [reply packages/jsts/tests/rules/index.test.ts:47]: it's already doing it, right?

File: .gitignore (hunk 1 above)
Comment (yassin-kammoun-sonarsource) [reply .gitignore:49]: 👍

File: packages/jsts/tests/rules/index.test.ts (hunk 2 above)
Comment (yassin-kammoun-sonarsource) [reply packages/jsts/tests/rules/index.test.ts:47]: Yes, sorry, I missed it 👍

**Pull request merged**